pkgs/top-level/all-packages.nix

```diff
@@ -1848,7 +1848,7 @@ with pkgs;
 
   hyperpotamus = callPackage ../tools/misc/hyperpotamus { };
 
-  immich-cli = callPackage ../tools/misc/immich-cli { };
+  immich-cli = immich.cli;
```

Comment:
IMO we are better off removing this by moving it to `aliases.nix` and giving a `throw` there

Reply:
No throw IMHO, just an alias is fine. People who don't like aliases should disallow them in their config.

Reply:
What I meant is, if it's a breaking change, users should know about it and be forced to replace `immich-cli` with `immich.cli`.

Reply:
Why would it be breaking?

Reply:
Because this switches from the legacy CLI (https://github.com/immich-app/cli) to the new CLI (https://github.com/immich-app/immich/tree/main/cli), from over a year ago. In [the PR for the new CLI](https://github.com/immich-app/immich/pull/3066) they mention wanting to wait for the new CLI to stabilise, so clearly some rewrite was involved here.

Reply:
I'm not entirely clear on the issue at hand right here, but to clarify: The legacy CLI will not work with any even remotely recent version of Immich.

Reply:
Actually on that topic, we should also have a release note about it.